src/python/bench/test_translate.py

```diff
@@ -0,0 +1,17 @@
+import sys
```

Comment:
Consider adding a short comment at the top of this file what this script does and a usage example.